### DISCUSSION TITLE
Fuzzy skin with paint

### DISCUSSION THREAD

**Discussion by ZombieZakPro** [OrcaSlicer/OrcaSlicer#4952] [Ideas]
Basically, you can draw with the support paint where the fuzzy skin will be.
Fuzzy skin is very hard to manage other ways this may help with making much more professional parts

But furthermore, this is a great slicer and made a lot of tasks much easier. This is only a suggestion :)

**Comment by Thynix** [comment]
For reference, the equivalent PrusaSlicer feature was added here https://github.com/prusa3d/PrusaSlicer/issues/6988

**Comment by confuzled-dev** [comment]
very intersted in this feature
